src/ng/rootScope.js

```diff
@@ -370,7 +372,9 @@ function $RootScopeProvider() {
               last: initWatchVal,
               get: get,
               exp: watchExp,
-              eq: !!objectEquality
+              eq: !!objectEquality,
+              equals: isFunction(objectEquality) ? objectEquality : equals,
+              copy: isFunction(copier) ? copier : copy
```

Comment:
if we decide to move forward, then this looks like the wrong approach. We should not add new properties, and change `get` and `listener` so they are new functions that do the custom compare and copy.

https://github.com/angular/angular.js/pull/10096#issuecomment-69366318

can be used as a template, but in this specific case, or can be simplified by using the `$watch` function closure.

With change, then there is no need to change any of the delegates